hugo/content/faqs/_index.md

```diff
@@ -2,87 +2,88 @@
 title: Frequently Asked Questions
 ---
 
-[faq]
-title = "FAQs"
+### FAQs
 
-## About the Genome Portal
+#### About the Genome Portal
 
-[[faq.section]]
-question = "What is the Swedish Reference Genome Portal?"
-answer = "TBD"
+{{< faq_block title="How does the Swedish Reference Genome Portal differ from other genome portal initiatives? (e.g., Ensembl, UCSC)" >}}
+The Swedish Reference Genome Portal (SRGP) does not replace existing genome portal initiatives, as it has specific aims and scope:
 
-[[faq.section]]
-question = "How does the Genome Portal differ from existing genome browsers such as Ensembl, USCS, JGI, IGV, among others?"
-answer = "TBD"
+- The SRGP targets the Swedish research community and is restricted to non-human eukaryotic species. Therefore, the species displayed here are typically limited to those submitted by researchers affiliated with Swedish institutions. In contrast, global genome portals such as Ensembl and UCSC have much broader taxonomic scope, aiming to include species from around the world, including humans and bacteria.
 
-[[faq.section]]
-question = "What are the benefits of using the Genome Portal?"
-answer = "TBD"
+- The SRGP's primary goal is to facilitate access, visualisation, and interpretation of genomic data, with a focus on offering a powerful genome browser. Global genome portals like Ensembl and UCSC offer additional features including those related to comparative genomics, besides genome browser capabilities.
```

Comment:
Could it be good to mention _how_ we facilitate access? I.e. by aggregating links to data in a single page.

Reply:
@brinkdp sounds good! Could you please suggest a commit?

Reply:
Sure! This is a quick edit that tries to fit that into the flow of the previous text. It's a perhaps a little rough, so feel free to make changes

```suggestion
- The SRGP's primary goal is to facilitate access, visualisation, and interpretation of genomic data, with a focus on offering a powerful genome browser. SRGP facilitates access and discovery by aggregating links to datasets associated with each genome assembly in a single page. Global genome portals like Ensembl and UCSC might, however, offer their own specific features besides their genome browser capabilities, such as support for comparative genomics analyses.